### PR TITLE
Code to run Web and App test in a single session.

### PR DESCRIPTION
**Pull request by SujaySawant**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <threads>5</threads>
+    </properties>
+
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/io.appium/java-client -->
+        <dependency>
+            <groupId>io.appium</groupId>
+            <artifactId>java-client</artifactId>
+            <version>7.5.1</version>
```
Comment (sanketsmali03) [inline pom.xml]: Add Versions under properties for all dependencies

File: README.md [hunk 2]
```diff
@@ -1 +1,118 @@
...
+    - [Wikipedia Sample App](https://www.browserstack.com/app-automate/sample-apps/android/WikipediaSample.apk)
+    - Chrome Mobile App
+- Apps used to test on iOS:
+    - [BStack Sample App](https://www.browserstack.com/app-automate/sample-apps/ios/BStackSampleApp.ipa)
+    - Safari Mobile App
+    - Chrome Mobile App
+- Export the environment variables for the Username and Access Key of your BrowserStack account.
+  ```sh
+  export BROWSERSTACK_USERNAME=<browserstack-username> && export BROWSERSTACK_ACCESS_KEY=<browserstack-access-key>
+  ```
```
Comment (sanketsmali03) [inline README.md]: Add this to the beginning

File: README.md [hunk 3]
```diff
@@ -1 +1,118 @@
-# Java MultiPlatform Scenarios
\ No newline at end of file
+# Java MultiPlatform Scenarios <a href="https://www.browserstack.com/"><img src="https://www.vectorlogo.zone/logos/browserstack/browserstack-icon.svg" alt="BrowserStack" height="30"/></a> <a href="https://java.com"><img src="https://www.vectorlogo.zone/logos/java/java-icon.svg" alt="Java" height="30" /></a> <a href="https://www.selenium.dev/"><img src="https://seeklogo.com/images/S/selenium-logo-DB9103D7CF-seeklogo.com.png" alt="Selenium" height="30" /></a>
+
+Automate a real end-to-end user flow between app and browser in a single session.
```
Comment (sanketsmali03) [inline README.md]: Can we add little more description that this repo is particular for app automate platform

File: pom.xml (hunk 1 above)
Comment (SujaySawant) [reply pom.xml]: Added Versions under properties for all dependencies

File: README.md (hunk 2 above)
Comment (SujaySawant) [reply README.md]: Added this to the beginning

File: README.md (hunk 3 above)
Comment (SujaySawant) [reply README.md]: Added proper description in README.

**Comment by sanketsmali03** [comment]
Changes approved

**Comment by SujaySawant** [comment]
Changes were not merged to the develop branch.

**Pull request merged**